core-aam/index.html

```diff
@@ -5711,6 +5711,48 @@ <h4>Not Mapped</h4>
               >. Its absence not only indicates that the accessible object is not grabbed, but further defines it as not grab-able. These cases are marked as "Not mapped" without an asterisk.
             </p>
           </section>
+          <h4 id="ariaActions"><code>aria-actions</code></h4>
+          <table aria-labelledby="ariaActions">
+            <tbody>
+              <tr>
+                <th>ARIA Specification</th>
+                <td>
+                  <a class="property-reference" href="#aria-actions"><code>aria-actions</code></a>
+                </td>
+              </tr>
+              <tr>
+                <th>MSAA + IAccessible2</th>
+                <td>
+                  <span class="property"
+                    >Interface: <code>IAccessibleAction</code>, using the accessible nodes matching `aria-actions` IDREFs, if the referenced objects are in the accessibility tree</span
+                  ><br />
+                  <span class="property">Object Attribute: <code>aria-actions</code> &lt;value&gt;</span>
+                </td>
+              </tr>
+              <tr>
+                <th><abbr title="User Interface Automation">UIA</abbr></th>
+                <td>
+                  <span class="property"
+                    >Custom Property: <code>AccessibleActions</code>: element array of accessible nodes matching IDREFs, if the referenced objects are in the accessibility tree</span
+                  ><br />
+                  <span class="property">Object Attribute: <code>aria-actions</code> &lt;value&gt;</span>
+                </td>
+              </tr>
+              <tr>
+                <th><abbr title="Accessibility Toolkit">ATK</abbr>/<abbr title="Assistive Technology-Service Provider Interface">AT-SPI</abbr></th>
+                <td>
+                  <span class="relation">Relation: <code>RELATION_DETAILS</code> points to accessible nodes matching IDREFs, if the referenced objects are in the accessibility tree</span><br />
+                  <span class="relation">Reverse Relation: <code>RELATION_DETAILS_FOR</code> points to element</span>
```

Comment:
Why aren't we using AtkAction here, similar to IA2?

Reply:
We should! I just wasn't familiar enough with ATK to know if that was the right mapping, so I left it with the earlier version until I could ask someone who was :D